php/class-asset-manager-scripts.php

```diff
@@ -93,38 +99,38 @@ public function set_asset_type_defaults() {
 	 * Add filters for managing async or defer load methods
 	 */
 	public function manage_async() {
-		add_filter( 'script_loader_tag', array( $this, 'add_attributes' ), 10, 2 );
-		add_filter( 'wpcom_js_do_concat', array( $this, 'disable_concat' ), 10, 2 );
-		add_filter( 'js_do_concat', array( $this, 'disable_concat' ), 10, 2 );
+		add_filter( 'script_loader_tag', [ $this, 'add_attributes' ], 10, 2 );
+		add_filter( 'wpcom_js_do_concat', [ $this, 'disable_concat' ], 10, 2 );
+		add_filter( 'js_do_concat', [ $this, 'disable_concat' ], 10, 2 );
 	}
 
 	/**
 	 * Add async and defer attributes to script tags where necessary
 	 *
-	 * @param string $tag    HTML <script> tag
-	 * @param string $handle Handle of script
+	 * @param string $tag    HTML <script> tag.
+	 * @param string $handle Handle of script.
 	 *
 	 * @return string The updated script tag
 	 */
 	public function add_attributes( $tag, $handle ) {
 		if ( isset( $this->assets_by_handle[ $handle ] ) ) {
 			$this_script = $this->assets_by_handle[ $handle ];
-			$attribute = 'async-defer' === $this_script['load_method'] ? 'async defer' : $this_script['load_method'];
+			$attribute   = 'async-defer' === $this_script['load_method'] ? 'async defer' : $this_script['load_method'];
 
 			if ( in_array( $handle, $this->async_scripts, true ) && ! preg_match( "/[^-_]{$attribute}[^-_]/", $tag ) ) {
-				// Insert load attribute in front of src attribute to ensure we don't add it to script tags containing inline code
-				$tag = str_replace( "src=", $attribute . " src=", $tag );
+				// Insert load attribute in front of src attribute to ensure we don't add it to script tags containing inline code.
+				$tag = str_replace( 'src=', $attribute . ' src=', $tag );
 			}
 		}
 
 		return $tag;
 	}
 
 	/**
-	 * Disable JS concatenation for async and defer scripts
+	 * Disable JS concatenation for async and defer scripts.
 	 *
-	 * @param bool $do_concat Whether or not to concatenate this script
-	 * @param string $handle  Handle for enqueued script
+	 * @param bool   $do_concat Whether or not to concatenate this script.
+	 * @param string $handle  Handle for enqueued script.
```

Comment:
Related to my other question :)

```suggestion
	 * @param bool   $do_concat Whether or not to concatenate this script.
	 * @param string $handle    Handle for enqueued script.
```